Readability.js

```diff
@@ -1616,6 +1618,38 @@ Readability.prototype = {
     }
   },
 
+  /* convert images and figures that have properties like data-src into images that can be loaded without JS */
+  _fixLazyImages: function (root) {
+    this._forEachNode(this._getAllNodesWithTag(root, ["img", "picture", "figure"]), function (elem) {
+      if ((!elem.src && (!elem.srcset || elem.srcset == "null")) || elem.className.toLowerCase().indexOf("lazy") !== -1) {
```

Comment:
Yeah, a comment here would be good. I'm sure the jsdom people would appreciate you filing a bug if you have time, and then we could reference that here. If not, let me know and I can do it.

Reply:
Good point, I've opened an issue here: https://github.com/jsdom/jsdom/issues/2580.